src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+        public bool bWarningCallback;
+        public bool bErrorCallback;
+        public int errorCount;
+        public int warningCount;
+        public bool WarningInnerExceptionSet = false;
+        public bool ErrorInnerExceptionSet = false;
```

Comment:
@supporterPL since this is a new source file prefer to follow the [coding-style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) so it serves as a template for any other new file on the same project.